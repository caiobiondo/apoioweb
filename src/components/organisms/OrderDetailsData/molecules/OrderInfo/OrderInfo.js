import React, { Component } from 'react';
import { FormattedMessage } from 'react-intl';
import { formatDate, formatTime, formatCurrency } from 'locale/utils';
import { orange100 } from 'styles/colors';
import { Paper } from 'natura-ui';

import SectionTitle from 'components/molecules/SectionTitle/SectionTitle';
import OrderDatum from '../OrderDatum/OrderDatum';

import {
  Wrapper,
  OrderInfos,
  OrderInfosColumn,
  OrderInfosRow,
  OrderData,
} from './OrderInfo.styles';
import { OrderDatumValue } from '../OrderDatum/OrderDatum.styles';

export default class OrderInfo extends Component {
  constructor(props) {
    super(props);
    this.state = {
      expanded: !props.importing,
    };
  }

  renderPaymentMethods = paymentMethods => {
    return paymentMethods.map((paymentMethod, index) => {
      return this.renderPaymentMethod(paymentMethod, index);
    });
  };

  renderPaymentMethod = ({ descricaoWeb, valorAcumulado }, index) => {
    const { intl } = this.props;

    return (
      <OrderDatumValue key={index}>
        {`${descricaoWeb} - ${formatCurrency(valorAcumulado, intl)}`}
      </OrderDatumValue>
    );
  };

  renderAddressStreet = ({ tipoLogradouro, nomeLogradouro, numero, bairro }) => {
    return `${tipoLogradouro} ${nomeLogradouro}, ${numero} - ${bairro}`;
  };

  renderAddressZipcode = ({ cep }) => {
    return ` ${cep.slice(0, 5)}-${cep.slice(5)}`;
  };

  renderAddress = address => {
    return (
      <OrderData>
        <OrderDatum type="long">
          <OrderDatumValue>{this.renderAddressStreet(address)}</OrderDatumValue>
          <OrderDatumValue>{`${address.cidade} - ${address.estado}`}</OrderDatumValue>
          <OrderDatumValue>
            <FormattedMessage id="orderDeliveryAddressZipcode" />:
            {this.renderAddressZipcode(address)}
          </OrderDatumValue>
        </OrderDatum>
      </OrderData>
    );
  };

  renderEvents = events => {
    return events.map(event => this.renderEvent(event));
  };

  renderEvent = ({ id, ocorrencia, motivo, solucao, data }) => {
    const { intl } = this.props;

    return (
      <OrderData key={id}>
        <OrderDatum type="long" label="orderEvents" value={ocorrencia} />
        <OrderDatum type="long" label="orderEventMotive" value={motivo} />
        <OrderDatum type="long" label="orderEventSolution" value={solucao} />
        <OrderDatum type="medium" label="orderEventDate" value={formatDate(data, intl, '-')} />
        <OrderDatum type="medium" label="orderEventHour" value={formatTime(data, intl, '-')} />
      </OrderData>
    );
  };

  renderOrderData = order => {
    const {
      codigoPedido,
      ciclo,
      dataPedido,
      status,
      quantidadeItens,
      pontos,
      meioCaptacao,
      envioBoleto,
    } = order;
    const { intl } = this.props;

    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderData" />
        <OrderData>
          <OrderDatum type="short" label="orderNumber" value={codigoPedido} />
          <OrderDatum type="short" label="orderCycle" value={ciclo} />
          <OrderDatum type="short" label="orderDate" value={formatDate(dataPedido, intl, '-')} />
          <OrderDatum type="long" label="orderStatus" value={status} />
          <OrderDatum type="short" label="orderItemsQuantity" value={quantidadeItens} />
          <OrderDatum type="short" label="orderTotalScore" value={pontos} />
          <OrderDatum type="short" label="orderReception" value={meioCaptacao} />
          <OrderDatum type="long" label="orderPaymentSlip" value={envioBoleto} />
        </OrderData>
      </OrderInfosRow>
    );
  };

  renderOrderValues = ({ valor, vlTaxaReal, qtdParcelas, valorParcela, vlAPagarSomaTaxa }) => {
    const { intl } = this.props;

    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderValues" />
        <OrderData>
          <OrderDatum
            type="short"
            label="orderDetailsOrderValue"
            value={formatCurrency(valor, intl)}
          />
          <OrderDatum type="short" label="orderTaxValue" value={formatCurrency(vlTaxaReal, intl)} />
          <OrderDatum type="short" label="orderInstallmentsNumber" value={qtdParcelas} />
          <OrderDatum
            type="short"
            label="orderInstallmentValue"
            value={formatCurrency(valorParcela, intl)}
          />
          <OrderDatum
            type="short"
            label="orderValue"
            value={formatCurrency(vlAPagarSomaTaxa, intl)}
          />
        </OrderData>
      </OrderInfosRow>
    );
  };

  renderOrderPaymentMethods = ({ formasPagamento }) => {
    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderPayment" />
        <OrderData>
          <OrderDatum type="long" label="orderPaymentMessage">
            {this.renderPaymentMethods(formasPagamento)}
          </OrderDatum>
        </OrderData>
      </OrderInfosRow>
    );
  };

  renderOrderReceipt = ({ numeroNotaFiscal, dataEmissaoNotaFiscal, modeloComercial }) => {
    const { intl } = this.props;

    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderReceipt" />
        <OrderData>
          <OrderDatum type="short" label="orderReceiptNumber" value={numeroNotaFiscal} />
          <OrderDatum
            type="short"
            label="orderReceiptEmitDate"
            value={formatDate(dataEmissaoNotaFiscal, intl, '-')}
          />
          <OrderDatum type="short" label="orderReceiptModel" value={modeloComercial} />
        </OrderData>
      </OrderInfosRow>
    );
  };

  renderOrderDelivery = ({ entrega }) => {
    const {
      dtPrevisaoEntrega,
      dtNovaPrevisaoEntrega,
      dtPrevisaoEntregaProrrogada,
      dtPrimeiraTentativaEntrega,
      dtSegundaTentativaEntrega,
      dtRealEntrega,
    } = entrega;
    const { intl } = this.props;

    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderDelivery" />
        <OrderData>
          <OrderDatum
            type="medium"
            label="orderDeliveryEstimatedDate"
            value={formatDate(dtPrevisaoEntrega, intl, '-')}
          />
          <OrderDatum
            type="medium"
            label="orderDeliveryNewEstimatedDate"
            value={formatDate(dtNovaPrevisaoEntrega, intl, '-')}
          />
          <OrderDatum
            type="medium"
            label="orderDeliveryExtendedEstimatedDate"
            value={formatDate(dtPrevisaoEntregaProrrogada, intl, '-')}
          />
          <OrderDatum
            type="medium"
            label="orderDeliveryFirstTryDate"
            value={formatDate(dtPrimeiraTentativaEntrega, intl, '-')}
          />
          <OrderDatum
            type="medium"
            label="orderDeliverySecondTryDate"
            value={formatDate(dtSegundaTentativaEntrega, intl, '-')}
          />
          <OrderDatum
            type="medium"
            label="orderDeliveryRealDate"
            value={formatDate(dtRealEntrega, intl, '-')}
          />
        </OrderData>
      </OrderInfosRow>
    );
  };

  renderOrderEvents = ({ ocorrencias }) => {
    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderEvents" />
        {this.renderEvents(ocorrencias)}
      </OrderInfosRow>
    );
  };

  renderOrderDeliveryAddress = ({ entrega: { enderecoEntrega } }) => {
    return (
      <OrderInfosRow>
        <SectionTitle color={orange100} value="orderDeliveryAddress" />
        {this.renderAddress(enderecoEntrega)}
      </OrderInfosRow>
    );
  };

  onToggleSize = () => {
    this.setState({ expanded: !this.state.expanded });
  };

  render() {
    const { order, importing } = this.props;
    const { expanded } = this.state;

    return (
      <Paper style={Wrapper}>
        <SectionTitle
          iconName="ico_info"
          value="orderInformations"
          expandable={importing}
          expanded={expanded}
          onToggle={this.onToggleSize}
        />
        {expanded && (
          <OrderInfos>
            <OrderInfosColumn>
              {this.renderOrderData(order)}
              {this.renderOrderValues(order)}
              {this.renderOrderPaymentMethods(order)}
              {this.renderOrderReceipt(order)}
            </OrderInfosColumn>

            <OrderInfosColumn>
              {this.renderOrderDelivery(order)}
              {this.renderOrderEvents(order)}
              {this.renderOrderDeliveryAddress(order)}
            </OrderInfosColumn>
          </OrderInfos>
        )}
      </Paper>
    );
  }
}
