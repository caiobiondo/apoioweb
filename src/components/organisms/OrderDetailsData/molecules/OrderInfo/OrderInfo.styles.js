import styled from 'styled-components';
import { Responsive } from '@entria/components';
import { gray150 } from 'styles/colors';
import { spMedium } from 'styles/spacing';

export const Wrapper = {
  padding: '30px',
  marginBottom: spMedium,
};

export const OrderInfos = styled.div`
  display: flex;

  @media (max-width: ${Responsive.VIEWPORT.MEDIUM}px) {
    flex-direction: column;
  }
`;

export const OrderInfosRow = styled.div`
  padding: 10px;

  & + & {
    border-top: 1px dotted ${gray150};
    padding-top: 36px;
    padding-bottom: 4px;
  }
`;

export const OrderInfosColumn = styled.div`
  flex: 1 1 60%;

  & + & {
    border-left: 1px solid ${gray150};
    flex-basis: 40%;

    ${OrderInfosRow} {
      padding-left: 65.5px;
      padding-right: 10px;
    }
  }

  @media (max-width: ${Responsive.VIEWPORT.MEDIUM}px) {
    & + & {
      border-left-width: 0;
      border-top: 1px dotted ${gray150};
      flex-basis: auto;

      ${OrderInfosRow} {
        padding-left: 10px;
        padding-top: 36px;
        padding-bottom: 4px;
      }
    }
  }
`;

export const OrderData = styled.div`
  display: flex;
  flex-wrap: wrap;
  font-family: Roboto-Regular;

  & + & {
    padding-top: 32px;
    border-top: 1px dotted ${gray150};
  }
`;
