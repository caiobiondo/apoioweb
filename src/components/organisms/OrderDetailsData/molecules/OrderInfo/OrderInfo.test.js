import React from 'react';
import { shallow } from 'enzyme';
import OrderInfo from './OrderInfo';
import orderData from '../../__mocks__/orderData.json';

const setup = propOverrides => {
  const props = Object.assign(
    {
      intl: {
        formatCurrency: value => `formatedCurrency ${value}`,
        formatDate: value => `formatedDate`,
        formatNumber: value => `formatedNumber ${value}`,
        formatTime: value => `formatedTime`,
      },
      order: orderData.order,
      importing: false,
    },
    propOverrides,
  );

  const result = shallow(<OrderInfo {...props} />);

  return {
    props,
    result,
  };
};

describe('OrderInfo', () => {
  it('renders order details info', () => {
    // given
    // when
    const { result } = setup();

    // then
    expect(result).toMatchSnapshot();
  });

  it('renders order details info (in importing mode - collapsed)', () => {
    // given
    // when
    const { result } = setup({ importing: true });

    // then
    expect(result).toMatchSnapshot();
  });
});
