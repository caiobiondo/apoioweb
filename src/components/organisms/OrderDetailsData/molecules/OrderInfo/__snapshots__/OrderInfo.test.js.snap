// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderInfo renders order details info (in importing mode - collapsed) 1`] = `
<Paper
  circle={false}
  rounded={true}
  style={
    Object {
      "marginBottom": "24px",
      "padding": "30px",
    }
  }
  transitionEnabled={true}
  zDepth={1}
>
  <SectionTitle
    iconName="ico_info"
    value="orderInformations"
  />
  <OrderInfostyles__OrderInfos>
    <OrderInfostyles__OrderInfosColumn>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderData"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderNumber"
            type="short"
            value={378325469}
          />
          <OrderDatum
            label="orderCycle"
            type="short"
            value="14/2017"
          />
          <OrderDatum
            label="orderDate"
            type="short"
            value="formatedDate"
          />
          <OrderDatum
            label="orderStatus"
            type="long"
            value="CONCLUÍDO E APROVADO"
          />
          <OrderDatum
            label="orderItemsQuantity"
            type="short"
            value={1}
          />
          <OrderDatum
            label="orderTotalScore"
            type="short"
            value={68}
          />
          <OrderDatum
            label="orderReception"
            type="short"
            value="WEB"
          />
          <OrderDatum
            label="orderPaymentSlip"
            type="long"
            value="Impresso e Enviado com o Pedido"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderValues"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderDetailsOrderValue"
            type="short"
            value="formatedNumber 390.0799865722656"
          />
          <OrderDatum
            label="orderTaxValue"
            type="short"
            value="formatedNumber 11.702399597167968"
          />
          <OrderDatum
            label="orderInstallmentsNumber"
            type="short"
            value={2}
          />
          <OrderDatum
            label="orderInstallmentValue"
            type="short"
            value="formatedNumber 200.89"
          />
          <OrderDatum
            label="orderValue"
            type="short"
            value="formatedNumber 401.78238616943355"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderPayment"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderPaymentMessage"
            type="long"
          >
            <OrderDatumstyles__OrderDatumValue
              key="0"
            >
              21/42 DIAS - formatedNumber 401.78238616943355
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue
              key="1"
            >
              21/42 DIAS - formatedNumber 401.78238616943355
            </OrderDatumstyles__OrderDatumValue>
          </OrderDatum>
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderReceipt"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderReceiptNumber"
            type="short"
            value={null}
          />
          <OrderDatum
            label="orderReceiptEmitDate"
            type="short"
            value="-"
          />
          <OrderDatum
            label="orderReceiptModel"
            type="short"
            value="Consultora"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
    </OrderInfostyles__OrderInfosColumn>
    <OrderInfostyles__OrderInfosColumn>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderDelivery"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderDeliveryEstimatedDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderDeliveryNewEstimatedDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryExtendedEstimatedDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryFirstTryDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliverySecondTryDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryRealDate"
            type="medium"
            value="formatedDate"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderEvents"
        />
        <OrderInfostyles__OrderData
          key="87"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Saiu para Entrega"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Saiu para Entrega"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="66"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="ENTREGA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="1a. Tentativa de Entrega Sem Sucesso. Motivo: Ausência no Ato da Entrega - Aguarde Nova Tentativa."
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="1a. Tentativa de Entrega Sem Sucesso - Ausência no Ato da Entrega - Aguarde Nova Tentativa."
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="49"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Chegada Filial Destino"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Chegada Filial Destino"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="88"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Coletado pela Transportadora"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Coletado pela Transportadora"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderDeliveryAddress"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            type="long"
          >
            <OrderDatumstyles__OrderDatumValue>
              R ACUCENA, 0 - XANGRI LA
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue>
              CONTAGEM - MG
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue>
              <FormattedMessage
                id="orderDeliveryAddressZipcode"
                values={Object {}}
              />
              :
               32187-090
            </OrderDatumstyles__OrderDatumValue>
          </OrderDatum>
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
    </OrderInfostyles__OrderInfosColumn>
  </OrderInfostyles__OrderInfos>
</Paper>
`;

exports[`OrderInfo renders order details info 1`] = `
<Paper
  circle={false}
  rounded={true}
  style={
    Object {
      "marginBottom": "24px",
      "padding": "30px",
    }
  }
  transitionEnabled={true}
  zDepth={1}
>
  <SectionTitle
    iconName="ico_info"
    value="orderInformations"
  />
  <OrderInfostyles__OrderInfos>
    <OrderInfostyles__OrderInfosColumn>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderData"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderNumber"
            type="short"
            value={378325469}
          />
          <OrderDatum
            label="orderCycle"
            type="short"
            value="14/2017"
          />
          <OrderDatum
            label="orderDate"
            type="short"
            value="formatedDate"
          />
          <OrderDatum
            label="orderStatus"
            type="long"
            value="CONCLUÍDO E APROVADO"
          />
          <OrderDatum
            label="orderItemsQuantity"
            type="short"
            value={1}
          />
          <OrderDatum
            label="orderTotalScore"
            type="short"
            value={68}
          />
          <OrderDatum
            label="orderReception"
            type="short"
            value="WEB"
          />
          <OrderDatum
            label="orderPaymentSlip"
            type="long"
            value="Impresso e Enviado com o Pedido"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderValues"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderDetailsOrderValue"
            type="short"
            value="formatedNumber 390.0799865722656"
          />
          <OrderDatum
            label="orderTaxValue"
            type="short"
            value="formatedNumber 11.702399597167968"
          />
          <OrderDatum
            label="orderInstallmentsNumber"
            type="short"
            value={2}
          />
          <OrderDatum
            label="orderInstallmentValue"
            type="short"
            value="formatedNumber 200.89"
          />
          <OrderDatum
            label="orderValue"
            type="short"
            value="formatedNumber 401.78238616943355"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderPayment"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderPaymentMessage"
            type="long"
          >
            <OrderDatumstyles__OrderDatumValue
              key="0"
            >
              21/42 DIAS - formatedNumber 401.78238616943355
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue
              key="1"
            >
              21/42 DIAS - formatedNumber 401.78238616943355
            </OrderDatumstyles__OrderDatumValue>
          </OrderDatum>
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderReceipt"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderReceiptNumber"
            type="short"
            value={null}
          />
          <OrderDatum
            label="orderReceiptEmitDate"
            type="short"
            value="-"
          />
          <OrderDatum
            label="orderReceiptModel"
            type="short"
            value="Consultora"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
    </OrderInfostyles__OrderInfosColumn>
    <OrderInfostyles__OrderInfosColumn>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderDelivery"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            label="orderDeliveryEstimatedDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderDeliveryNewEstimatedDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryExtendedEstimatedDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryFirstTryDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliverySecondTryDate"
            type="medium"
            value="-"
          />
          <OrderDatum
            label="orderDeliveryRealDate"
            type="medium"
            value="formatedDate"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderEvents"
        />
        <OrderInfostyles__OrderData
          key="87"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Saiu para Entrega"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Saiu para Entrega"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="66"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="ENTREGA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="1a. Tentativa de Entrega Sem Sucesso. Motivo: Ausência no Ato da Entrega - Aguarde Nova Tentativa."
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="1a. Tentativa de Entrega Sem Sucesso - Ausência no Ato da Entrega - Aguarde Nova Tentativa."
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="49"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Chegada Filial Destino"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Chegada Filial Destino"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
        <OrderInfostyles__OrderData
          key="88"
        >
          <OrderDatum
            label="orderEvents"
            type="long"
            value="INFORMATIVA"
          />
          <OrderDatum
            label="orderEventMotive"
            type="long"
            value="Coletado pela Transportadora"
          />
          <OrderDatum
            label="orderEventSolution"
            type="long"
            value="Coletado pela Transportadora"
          />
          <OrderDatum
            label="orderEventDate"
            type="medium"
            value="formatedDate"
          />
          <OrderDatum
            label="orderEventHour"
            type="medium"
            value="formatedTime"
          />
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
      <OrderInfostyles__OrderInfosRow>
        <SectionTitle
          color="#f3971f"
          value="orderDeliveryAddress"
        />
        <OrderInfostyles__OrderData>
          <OrderDatum
            type="long"
          >
            <OrderDatumstyles__OrderDatumValue>
              R ACUCENA, 0 - XANGRI LA
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue>
              CONTAGEM - MG
            </OrderDatumstyles__OrderDatumValue>
            <OrderDatumstyles__OrderDatumValue>
              <FormattedMessage
                id="orderDeliveryAddressZipcode"
                values={Object {}}
              />
              :
               32187-090
            </OrderDatumstyles__OrderDatumValue>
          </OrderDatum>
        </OrderInfostyles__OrderData>
      </OrderInfostyles__OrderInfosRow>
    </OrderInfostyles__OrderInfosColumn>
  </OrderInfostyles__OrderInfos>
</Paper>
`;
